src/AppInstallerCLICore/Workflows/WorkflowBase.cpp

```diff
@@ -426,6 +472,24 @@ namespace AppInstaller::CLI::Workflow
             AICLI_TERMINATE_CONTEXT(APPINSTALLER_CLI_ERROR_EXPERIMENTAL_FEATURE_DISABLED);
         }
     }
+
+    void SearchSourceUsingManifest(Execution::Context& context)

```

Comment:
>SearchSourceUsingManifest [](start = 9, length = 25)

I'm not sure that this is going to work out terribly well as implemented.

I would start with a similar search pattern as we correlate with elsewhere; the system reference strings.  Then fall back to the Id / Name combo.  And I would suggest Id to be CaseInsensitive as the match type.

If you want, you could wait to see how I'm doing it in list.